src/bridge_gui.cpp

```diff
@@ -137,9 +139,11 @@ class BuildBridgeWindow : public Window {
 	 */
 	StringID GetBridgeSelectString(const BuildBridgeData &bridge_data) const
 	{
-		SetDParam(0, bridge_data.spec->material);
-		SetDParam(1, PackVelocity(bridge_data.spec->speed, static_cast<VehicleType>(this->transport_type)));
-		SetDParam(2, bridge_data.cost);
+		const BridgeSpec* b = this->bridges->at(i).spec;
+
+		SetDParam(0, b->material);
+		SetDParam(1, b->speed);
```

Comment:
You've lost the PackVelocity which allows the game to show bridge speed limits with land or nautical velocity units.